app/pages/profile/feed.cjsx

```diff
@@ -28,7 +28,10 @@ module.exports = React.createClass
       comments: null
       error: null
     }, =>
-      talkClient.type('comments').get({user_id: user.id, page: page, sort: '-created_at'})
+      criteria = {user_id: user.id, page: page, sort: '-created_at'}
+      if @props.project?
+        criteria.project_id = @props.project.id
```

Comment:
Comments aren't filtered by project.  As previously mentioned, you can't filter by `project_id`, only **`section`**.  e.g. `"project-#{@props.project.id}"`